0.2/ssh_connection.py

```diff
@@ -0,0 +1,44 @@
+import paramiko
+from credentials import Credentials
+
+
+class SSHConnection:
+    def __init__(self, service, username, server, port):
+        self.service = service
+        self.username = username
+        self.server = server
+        self.port = port
+        self.cred = Credentials()
+        self.set_up_ssh_client()
+
+    def close(self):
+        self.sftp.close()
+        self.ssh.close()
+
+
```

Comment:
PEP8 says to only use one line between methods within a class. Check the other files as well